src/bodies/box2d_area.h

```diff
@@ -0,0 +1,73 @@
+#pragma once
+
+#include <godot_cpp/classes/physics_server2d.hpp>
+#include <godot_cpp/templates/hash_set.hpp>
+#include <godot_cpp/templates/vector.hpp>
+#include <godot_cpp/variant/callable.hpp>
+#include <godot_cpp/variant/variant.hpp>
+
+#include "../spaces/box2d_space.h"
+#include "box2d_collision_object.h"
+
+using namespace godot;
+
+class Box2DBody;
+
+class Box2DArea : public Box2DCollisionObject {
+	bool monitorable = false;
+	Callable monitor_callback;
+	Callable area_monitor_callback;
+	struct Gravity {
+		PhysicsServer2D::AreaSpaceOverrideMode override_mode = PhysicsServer2D::AreaSpaceOverrideMode::AREA_SPACE_OVERRIDE_DISABLED;
+		real_t gravity = 9.8;
+		b2Vec2 vector = b2Vec2(0, 1);
+		real_t is_point = false;
```

Comment:
This should be a `bool` (the actions you added caught this!)

Reply:
fixed.